dopamine/experiment5/experiment5_varplots.jl

```diff
@@ -0,0 +1,176 @@
+# VARIANCE IN SPEED PLOTS
+# Plots of wild type and mutant on bacterial lawn (Omura)
+
+using DataFrames
+using CSV
+using CategoricalArrays
+using GLMakie
+using Statistics
+using StatsBase
+
+
+
+# LOAD SPEEDS CSV INTO DATAFRAME
+experimentdir = @__DIR__
+
+speeds = DataFrame(CSV.File(joinpath(experimentdir, "speeds.csv")))
+
+# CALCULATE MEAN AND STD SPEED OF EACH TRACK
+tracks = groupby(speeds, [:experiment, :id, :track])
+trackstats = combine(tracks, :speed => mean => :meanspeed, :speed => std => :stdspeed)
+
+# FILTER TRACKS WITH NAN VALUES FOR STD
+filter!(:stdspeed => x -> !(isnan(x)), trackstats)
+
+# BIN TRACKS ACCORDING TO MEAN SPEED
+bin = Vector{Int}()
+
+for row in 1:nrow(trackstats)
+    if 0 ≤ trackstats.meanspeed[row] < 50
+        push!(bin, 1)
+    elseif 50 ≤ trackstats.meanspeed[row] < 100
+        push!(bin, 2)
+    elseif 100 ≤ trackstats.meanspeed[row] < 150
+        push!(bin, 3)
+    elseif 150 ≤ trackstats.meanspeed[row] < 200
+        push!(bin, 4)
+    elseif 200 ≤ trackstats.meanspeed[row] < 250
+        push!(bin, 5)
+    elseif 250 ≤ trackstats.meanspeed[row] < 300
+        push!(bin, 6)
+    elseif 300 ≤ trackstats.meanspeed[row]
+        push!(bin, 7)
+    end
+end
```

Comment:
```suggestion
bin = map(eachrow(trackstats)) do row
    count(row.meanspeed) do s
        s .> 0:50:300
    end
end
```

Boolean values can be summed (false is 0, true is 1), so if you count the number of "truths" for greater than ops, you get the same values you see here. Eg

```
julia> count(73 .> 0:50:300)
2

julia> count(220 .> 0:50:300)
5

julia> count(280 .> 50:50:300)
6
```

You could also put `(maximum(trackstats.meanspeed) + 50)` instead of 300 if you wanted it to be able to expand if there were more bins, or do something like `range(0, maximum(trackstats.meanspeed); length=7)` if you want to control the number of bins but let it be flexible with different ranges of values.